src/app/pages/CpsPgl/index.test.jsx

```diff
@@ -145,22 +144,22 @@ describe('CPS PGL Page', () => {
 
   it('should only render firstPublished timestamp for Igbo when lastPublished is less than 1 min later', async () => {
     const pageData = await preprocessor(
-      igboPageData,
+      pglAboutData,
```

Comment:
`igboPageData` is an  STY asset not PGL. This change ensures the ATI logic for STY's isn't used in a PGL test.